otter/test/test_scheduler.py

```diff
@@ -79,31 +94,32 @@ def _start_service(self):
         self.scheduler_service.running = True
 
     def test_start_service(self):
-        """
-        startService() calls super's startService() and creates SetPartitioner object
-        """
```

Comment:
sorry for that short comment, to elaborate:

> Unless the entire docstring fits on a line, place the closing quotes on a line by themselves. This way, Emacs' fill-paragraph command can be used on it. 

Additionally, having the starting line of a docstring come on the next line after the opening quotes _is_ conformant: 

>  The summary line may be on the same line as the opening quotes or on the next line. 

So I don't understand why we need to change all these docstrings.